test_filter.py

```diff
@@ -121,5 +121,5 @@ def test_filtered_adapter_invalid_filepath(self):
         adapter = casbin.persist.adapters.FilteredAdapter("examples/does_not_exist_policy.csv")
         e = casbin.Enforcer("examples/rbac_with_domains_model.conf", adapter)
 
-        with self.assertRaises(FileNotFoundError):
-            e.load_filtered_policy(None)
\ No newline at end of file
+        with self.assertRaises(RuntimeError):
```

Comment:
Why change this?

Reply:
It fixes a fail test that also happens on the master branch.
I'm also not sure as well why this test_filter.py file is on the root level rather than inside the tests/ folder, but if you don't see any objection I can move it there as well

Reply:
I didn't see any failed tests in our CI:

![image](https://user-images.githubusercontent.com/3787410/110965254-f8575600-838e-11eb-9f28-a531d6fd89ce.png)

Reply:
Yes, this is coming from the fact that this file test_filter.py is not inside the tests/ folder but outside.

The CI is only running on files inside /tests:
`coverage run -m unittest discover -s tests -t tests`

Reply:
OK